src/cljs/nr/lobby.cljs

```diff
@@ -164,7 +164,9 @@
                               :type "checkbox"
                               :on-change #(on-change-format-visibility slug %)
                               :checked (format-visible? slug)}]
-     [:label {:for id} (-> slug slug->format tr-format)]]))
+     [:label {:for id
+              :on-click #((.stopPropagation %))}
```

Comment:
```suggestion
              :on-click #(.stopPropagation %)}
```
I don't think you meant to call the return value of `stopPropogation`, right?

Reply:
I did not! Fixed.